gamefixes-steam/219990.py

```diff
@@ -14,5 +14,7 @@ def main():
     if os.path.isfile(os.path.join(os.getcwd(), 'GrimInternals64.exe')):
         util.replace_command('Grim Dawn.exe', 'GrimInternals64.exe')
 
-    # Fixes a black screen being rendered:
+    
+    """ Fix black screen """
```

Comment:
Prefer to use a regular comment by denoting a # symbol instead, since it's only one statement -- `util.protontricks('d3dcompiler_43')`.

Along with that change, if you remove the subprocess module too, I'll merge this ASAP

Reply:
Alright. I also changed the other comment to be homogeneous.